src/server/array_func.c

```diff
@@ -535,6 +535,24 @@ update_subjob_state(job *pjob, int newstate)
 	}
 	chk_array_doneness(parent);
 }
+/**
+ * @brief
+ * 		get_subjob_discarding - return the discarding flag of a subjob given by the parent job
+ * 		and integer index into the table for the subjob
+ *
+ * @param[in]	parent - pointer to the parent job
+ * @param[in]	iindx - first number of range
+ *
+ * @return	status
+ * @retval	-1	-  error
+ */
+int
+get_subjob_discarding(job *parent, int iindx)
+{
+	if (iindx == -1)
```

Comment:
Is there a scenario that iindx can be -1? If not then this function isn't really needed because caller can just access discarding variable of the subjob.

Reply:
Yes, function subjob_index_to_offset can make it -1.